docs/source/gbd2017_models/risks/low_birthweight_short_gestation/index.rst

```diff
@@ -51,3 +49,169 @@ described below, and relative to the joint TMREL.
+
+Causes that are affected by LBWSG
+---------------------------------
+
++----------+---------------------------------------------------------+
+| Cause id | Cause                                                   | 
++==========+=========================================================+
+|  302     | diarrheal diseases                                      |
++----------+---------------------------------------------------------+
+|  322     | lower respiratory tract infections                      |
++----------+---------------------------------------------------------+
+|  328     | upper respiratory tract infections                      | 
++----------+---------------------------------------------------------+
+|  329     | otitis media                                            |
++----------+---------------------------------------------------------+
+|  333     | pneumococcal meningitis                                 |
++----------+---------------------------------------------------------+
+|  334     | H influenzae type B meningitis                          |
++----------+---------------------------------------------------------+
+|  335     | meningococcal meningitis                                |
++----------+---------------------------------------------------------+
+|  336     | other meningitis                                        |
++----------+---------------------------------------------------------+
+|  337     | encephalitis                                            |
++----------+---------------------------------------------------------+
+|  381     | neonatal preterm birth complications                    |
++----------+---------------------------------------------------------+
+|  382     | neonatal encephalopathy due to birth asphyxia and trauma|
++----------+---------------------------------------------------------+
+|  383     | Neonatal sepsis and other neonatal infections           |
++----------+---------------------------------------------------------+
+|  384     | hemolytic disease and other neonatal jaundice           |
++----------+---------------------------------------------------------+
+|  385     | other neonatal disorders                                |
++----------+---------------------------------------------------------+
+|  686     | sudden infant death syndrome                            |
++----------+---------------------------------------------------------+
+
+Restrictions
+++++++++++++
+
+Assumptions and limitations
++++++++++++++++++++++++++++
+
+Because the relative risks from GBD are for all cause mortality in the early and late neonatal period, we will apply the following equation to calculate all mortality in the early and late neonatal age groups
+
+The all cause mortality rate (ACMR) is comprised of the sum of: 
+
+   - causes that are unaffected by lbwsg 
+   - causes that are affected by lbwsg but are NOT modelled (due to some reason we cannot model them)
```

Comment:
"causes that are affected by lbwsg but are not modeled in our vivarium simulation"

It could be good to have a little section about causes that are necessary to model in vivarium and not necessary when we are using LBWSG. Briefly, when we model LBWSG as a risk factor in our vivarium simulations, because we are modeling LBWSG to affect ACMR and not CSMRs, we do not need to individually model causes that are affected by LBWSG (these are our "affected and unmodeled causes"). We only need to explicitly model causes that are affected by other risk factors (or interventions) in our model - these will be our "affected and modeled causes."

Additionally... do we need a separate category of unaffected modeled causes? It seems unlikely but might be good to include (maybe could be a to-do note for now)

Reply:
ahh...ok so the EMRs_state are for the affected-modelled causes that are in the model and affected by risk factors other than lbwsg in our model. I originally thought they were the Lbwsg affected causes (but we are not modelling them!)

Reply:
Right. For instance, neonatal disorders is an affected unmodeled cause and LRI is an affected modeled cause. We didn't model neonatal disorders because it was ONLY affected by LBWSG and no other risks in our model. However, we did model LRI because it was affected by LBWSG AND the vitamin A deficiency risk factor.

Reply:
I made a table that hopefully illustrates this visually